changelog/18.0/18.0.0/summary.md

```diff
@@ -116,6 +117,35 @@ VTGate flag:
 
 - `--schema_change_signal_user` is deprecated and will be removed in `v19.0`
 
+#### <a id="deleted-flags"/>Deleted Command Line Flags
+
+Flags in `vtcombo`:
+- `--vtctld_addr`
+
+Flags in `vtctldclient`:
+- `--skip-preflight`
```

Comment:
This flag is specific to the ApplySchema command and should be documented that way. It's not a general vtctldclient flag.
Try `vtctldclient --help` vs `vtctldclient ApplySchema --help` to see the difference.

Reply:
Fixed via 05ac2000bd

Reply:
I'm assuming we mean skip_preflight as well? That's the one that I see mentioned in v17:
```
changelog/17.0/17.0.0/changelog.md: * ApplySchema: deprecate '--skip_preflight' flag [#10716](https://github.com/vitessio/vitess/pull/10716)
```

Reply:
I have added another section below for `skip_preflight`